styles.css

```diff
@@ -96,12 +109,12 @@ body {
   border-radius: 16px;
 }
 
-.work > h3 {
+.work h3 {
   font-size: 32px;
   line-height: 44px;
 }
 
-.work > ul {
+.work ul {
   margin: 0;
   padding: 0;
   display: flex;
```

Comment:
- [ ] It is noticed that `display: flex` property is repeated multiple times, Kindly let us stick to the DRY principle by not repeating a property more than it should 🙂. This can be done by having a class called `.flex-row` or `.flex-column` and assigning it wherever possible as a class name in your `html` markup code.
```
.flex-row {
  display: flex;
}

.flex-column {
  display: flex;
  flex-direction: column;
}
```
- And kindly attend to any repetitions in your code 🙂.